feature-libs/cart/quick-order/root/config/quick-order-form-config.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { Config } from '@spartacus/core';
+
+@Injectable({
+  providedIn: 'root',
+  useExisting: Config,
+})
+export abstract class QuickOrderFormConfig {
+  quickOrderForm?: {
+    displayProductImages: boolean;
+    maxProducts: number;
+    minCharactersBeforeRequest: number;
+  };
+}
```

Comment:
How about making this config a bit more nested, but more future-proof? If we would like for example to add maybe option in next release to configure max number of products in quick order list, we could another property to `quickOrder` instead of adding another top level config option.

```suggestion
export abstract class QuickOrderFormConfig {
  quickOrder?: {
    searchForm?: {
      displayProductImages: boolean;
      maxProducts: number;
      minCharactersBeforeRequest: number;
    }
  };
}
```

Reply:
agree